upgrade/src/constants.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
Do you think it makes sense to move all these parameters to `.env` file as well? Sorry that I forgot to mention this in the issue description.

Reply:
Sure, I moved them to `.env` file https://github.com/poanetwork/tokenbridge-contracts/pull/337/commits/f1aba7ffbd778ad927c731b90a568dfb14e2a11a